faq/blockstorage.mdx

```diff
@@ -52,4 +52,20 @@ The Block Storage billing scheme is not the same as for Object Storage. You are
 
 ## What other Scaleway resources are compatible with Block Storage?
 
-Block Storage volumes are compatible with [Scaleway Instances](/compute/instances/), [Managed Dabatase for PostgreSQL and MySQL](/managed-databases/postgresql-and-mysql/), and [Kubernetes](/containers/kubernetes/) Kapsule clusters.
\ No newline at end of file
+Block Storage volumes are compatible with [Scaleway Instances](/compute/instances/), [Managed Dabatase for PostgreSQL and MySQL](/managed-databases/postgresql-and-mysql/), and [Kubernetes](/containers/kubernetes/) Kapsule clusters.
+
+## How can I convert a Block Storage legacy volume into a Block Storage Low Latency volume?
+
+For better performance, you can convert your Block Storage legacy volumes into Block Storage Low Latency volumes.
+
+To convert your volume you need to:
+
+- Export the data: copy or back up the data from the Block storage legacy volume to an external location or intermediary storage (e.g., another temporary volume).
+
+- Create a Block Storage Low Latency volume
+
+- Import the data: restore or transfer the data from the backup or intermediary storage into the newly created volume.
```

Comment:
I find it weird to split the create and import in 2 steps whereas it is actually a single call performing both

Also should we not say this is a temporary/not ideal solution before being effectively able to change the storage class of a volume while preserving its id ?